agent/engine/engine_integ_test.go

```diff
@@ -284,3 +283,29 @@ func TestStartStopWithCredentials(t *testing.T) {
 	_, ok := credentialsManager.GetTaskCredentials(credentialsIDIntegTest)
 	assert.False(t, ok, "Credentials not removed from credentials manager for stopped task")
 }
+
+func TestContainerHealthCheck(t *testing.T) {
+	taskEngine, done, _ := setupWithDefaultConfig(t)
+	defer done()
+
+	stateChangeEvents := taskEngine.StateChangeEvents()
+
+	taskArn := "testTaskWithHealthCheck"
+	testTask := createTestHealthCheckTask(taskArn)
+
+	go taskEngine.AddTask(testTask)
+
+	verifyContainerRunningStateChange(t, taskEngine)
+	verifyTaskIsRunning(stateChangeEvents, testTask)
```

Comment:
i think we want to use [verifyTaskRunningStateChange](https://github.com/aws/amazon-ecs-agent/blob/1b9439dda781fd8aa4b5f6e5750b23774f7a930a/agent/engine/engine_integ_test.go#L107) here.

Reply:
`verifyTaskIsRunning` does the same state change event check as `verifyTaskRunningStateChange` and more like checking the task's ARN. So it would be preferable to use the former.